MapboxCoreNavigation/NavigationLocationManager.swift

```diff
@@ -33,5 +33,6 @@ open class NavigationLocationManager: CLLocationManager {
         }
         
         desiredAccuracy = kCLLocationAccuracyBestForNavigation
+        distanceFilter = kCLDistanceFilterNone
```

Comment:
`kCLDistanceFilterNone` is the default value of `distanceFilter`, so this probably has no effect.